01week/datatypes.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+//function that calls current date and time
+let displayDate = function(){
+  let date = new Date();
+  console.log("1. " + date);
+}
+
+//convert number to string
+let numberToString = function(x){
+  if(typeof(x)==='number'){
+    x = x.toString();
+    console.log("2. " + x + " is now a string. typeof(" + x + ")=" + typeof(x));
+  }
+}
+
+//convert string to number
+let stringToNumber = function(y){
+  if(typeof(y)==='string'){
+    y = Number(y);
+    console.log("3. " + y + " is now a number. typeof(" + y + ")=" + typeof(y));
+  }
+}
+
+//display datatype of input
+let whatDataType = function(z){
```

Comment:
What will happen when you run the following examples? what would you expect the answer to be? 
```
whatDataType(NaN);
whatDataType(null);
```

Reply:
I expect the answer to be:

whatDataType(NaN):
NaN has a datatype of number.

whatDataType(null):
null has a datatype of object.

...I ran the program, and verified that I am correct.